README.md

```diff
@@ -1,6 +1,6 @@
 ![com github qarmin czkawka](https://user-images.githubusercontent.com/41945903/102616149-66490400-4137-11eb-9cd6-813b2b070834.png)
 
-**Czkawka** is a simple, fast and easy to use app to remove unnecessary files from your computer.
+**Czkawka** (_pron. tch-kav-ka_, hiccup) is a simple, fast and easy to use app to remove unnecessary files from your computer.
```

Comment:
I suggest to not shorten *pronunciation* (or *pronounced*?) here. As a non-native English speaker, the meaning wasn't obvious.